R/fct_workflow_status.R

```diff
@@ -8,10 +8,12 @@
 #' @import tibble
 #'
 #' @noRd
-
+source("config/config.R")
+host_url <- host_url
 
 workflow_status <- function(report){
-  server <- rpecanapi::connect("http://141.142.217.168/", "carya", "illinois")
+  library(rpecanapi)
+  server <- rpecanapi::connect(host_url, "carya", "illinois")
```

Comment:
First, there's no need for the library call since you're calling the function by namespace. Second, you really don't want library loads within a function